app/lib/data/cubit/accounts_cubit.dart

```diff
@@ -6,13 +7,15 @@ part 'accounts_state.dart';
 
 class AccountsCubit extends Cubit<AccountsState> {
   IsarRepo repository = IsarRepo.instance;
+  List<Account> accounts = [];
```

Comment:
Is there a good reason why we should store the accounts here?

Reply:
Yes. Firstly, my goal here was to ensure error states don't replace the entire UI just to communicate an error. Inorder to achieve that the account list has to be persisted across states and putting it there does just that. 

An even better placement would be in the states themselves as a payload but that requires alot more changes than the scope of this PR allows.

Reply:
Alright, makes sense.